docs/corpora_schema.md

```diff
@@ -34,100 +35,110 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
 
 **Field name**|**Constraints**
 :--|:--
-tissue\_ontology\_term\_id|UBERON term
+tissue\_ontology\_term\_id|UBERON term. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay\_ontology\_term\_id|EFO term
 disease\_ontology\_term\_id|MONDO term or [PATO:0000461](http://bioportal.bioontology.org/ontologies/PATO?p=classes&conceptid=PATO%3A0000461)
 cell\_type\_ontology\_term\_id|CL term
 ethnicity\_ontology\_term\_id|HANCESTRO term, "na" if non-human
 development\_stage\_ontology\_term\_id|HsapDv term if human, child of EFO:0000399 otherwise
 
-The `tissue_ontology_term_id` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
+cellxgene requires ontology terms to enable search, comparison, and integration of data. When no appropriate ontology value is available, then the most
+precise accurate term MUST be used. For example if the `cell_type` field describes a relay interneuron, but the most specific available term in the CL
+ontology is CL:0000099 ("Interneuron"), then the interneuron term can be used to fulfill this requirement, and ensures that users searching for "neuron"
+are able to find these data. Users will still be able to access more specific cell type annotations that have been submitted with the
+data (but aren't required by the schema). A dataset comprising cells of the human embryo provides a more extreme example. In this case, the most  
+precise accurate term may be the root of the cell ontology `cell`, or its child term `cell in vitro`. The Cell Ontology is expanding over time, and we hope to
+migrate datasets to more defined terms as they are defined. In the mean time, having these less precise terms maximizes the findability (and therefore reusability)  
+of datasets.
 
-The `ontology_term_id` fields may be empty strings when no appropriate ontology value is available, for example if the
-`cell_type` field describes a cell type for which no ontology term exists or if the `ethnicity` is unknown. If the
-field is not empty, then it must be an [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning it is a CURIE
-where the prefix identifies the ontology. For example `EFO:0000001` is a term in the `EFO` ontology.
+#### Gene Metadata
 
-If the features of the dataset are human genes, then the feature ids must be [HGNC](https://www.genenames.org/about/guidelines/#!/#tocAnchor-1-7) approved
-gene symbols.
+Cellxgene uses standard gene symbols to ensure that all datasets it stores measure the same features and can therefore be integrated.
+If the samples being measured are human, then the feature ids MUST be [HGNC](https://www.genenames.org/about/guidelines/#!/#tocAnchor-1-7) approved
+gene symbols. If the features are mouse genes, then the feature ids SHOULD be [MGI](http://www.informatics.jax.org/mgihome/nomen/gene.shtml) gene symbols.
+For other organisms, gene symbols SHOULD be the accepted standard human readable symbols for that organism.
```

Comment:
I'm not sure what _accepted standard human readable symbols for that organism_ means or entails for validation.

Reply:
As a SHOULD, it means nothing for validation. This is guidance for users submitting data from organisms which we don't maintain stronger recommendations.